new-lamassu-admin/src/pages/Maintenance/Wizard/Wizard.js

```diff
@@ -72,27 +72,39 @@ const Wizard = ({ machine, cashoutSettings, locale, onClose, save, error }) => {
     {
       type: 'cashbox',
       schema: Yup.object().shape({
-        wasCashboxEmptied: Yup.string().required()
+        wasCashboxEmptied: Yup.string().required('Select one option.')
       }),
       cashoutRequired: false
     },
     {
       type: 'cassette 1',
       schema: Yup.object().shape({
         cassette1Count: Yup.number()
-          .required()
-          .min(0)
-          .max(CASHBOX_DEFAULT_CAPACITY)
+          .required('Bill count is a required.')
+          .min(
+            0,
+            ({ min }) => `Bill count must be greater than or equal to ${min}.`
+          )
+          .max(
+            CASHBOX_DEFAULT_CAPACITY,
+            ({ max }) => `Bill count must be less than or equal to ${max}.`
+          )
```

Comment:
I think that for min, max and required the YUP default message should be enough.